src/styles/MuiThemeProvider.js

```diff
@@ -80,8 +80,6 @@ export const MUI_SHEET_ORDER = [
   'MuiCardHeader',
   'MuiCard',
 
-  'MuiTextFieldLabel',
```

Comment:
We gonna need the `MuiFormHelperText` into that array to guarantee the override order.